src/libraries/adaptative-curve/ConstantsLib.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+library ConstantsLib {
```

Comment:
not sure if we need a constantLib for the IRM. Everything in the same file is great

Reply:
This and if we do follow [this suggestion](https://github.com/morpho-org/morpho-blue-irm/pull/66#pullrequestreview-1729338646) then there should be only 2 constants left in this lib and those would almost never be useful to integrators (who cares about the max curve steepness, you only really care about the curve steepness of the IRM you are interacting with)

Reply:
The same arguments could be used for Blue but still we extracted constants to a file because it's shown to be handy for our test suite alone ; having constants defined in IRM (such as the min/max rates) require to deploy the contract to test things using these bounds... For example what we do in ExpLib but thanks to the constants library we don't need to deploy an IRM to test the upper value of wExp

Reply:
I'm not saying that we should never extract constants. My point is that min/max rates should really be immutables, and I think that the other constants would almost never be useful

Reply:
What if there's a usecase to test something against the max curve steepness?
It doesn't seem to be a huge deal to move constants out to a library (if it's not for integrators, simply just for us)

Reply:
To me the benefit of having constants defined in a dedicated library and being able to access it outside of the IRM outweighs the loss of conciseness (now the IRM depends on 1 more file)

Whether min/max rates should be immutable is another topic that I'm open to discuss in a dedicated issue

Reply:
opened https://github.com/morpho-org/morpho-blue-irm/issues/96

I think that those topic are not so independent, but I'm fine with the current code in any case